src/index.ts

```diff
@@ -91,6 +91,7 @@ const main = async () => {
     for (const imageFile of imageFiles) {
       checkImageSize(`images/${chainIdentifier}/${imageFile}`);
     }
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
You should delete this commnet

Reply:
https://typescript-eslint.io/rules/no-explicit-any/